openedx_external_enrollments/external_enrollments/mit_hz_external_enrollment.py

```diff
@@ -70,6 +70,7 @@ def _get_bearer_token(self):
                 return 'missing token'
 
             return response.json()['access_token']
+        return ''
```

Comment:
Is it right to return an empty string? @luismorenolopera

Reply:
Yes, that's fine. Later it detects that it is not a valid token. But I think it is better to add a more descriptive message to this string @ivanvgh